src/components/footer/Footer.tsx

```diff
@@ -23,6 +25,16 @@ const FooterContainer = styled(Grid)`
     background-color: ${(props) => props.theme.colors.black};
 `;
 
+const MenuList = styled.ul`
```

Comment:
????

Reply:
Kan default ul worden. Had er eerst styling in staan maar niet meer verwijderd.